src/gtfs/mod.rs

```diff
@@ -552,6 +563,35 @@ struct Route {
     sort_order: Option<u32>,
 }
 
+#[derive(Default)]
+/// To associate a list of equipment with a stop
+pub struct EquipmentList {
```

Comment:
I would also gate that `struct` behind `#[cfg(all(feature = "gtfs", feature = "parser"))]` (and the `impl` block below).

Reply:
I put the `EquipmentList` back in `read.rs`